webpack.js

```diff
@@ -8,6 +8,17 @@ webpackConfig.module.rules[jsRule].exclude = BabelLoaderExcludeNodeModulesExcept
 	'@nextcloud/event-bus',
 	'semver',
 ])
+webpackConfig.module.rules[jsRule].options = {
+	plugins: ['add-module-exports'],
+	presets: [
+		/**
+		 * From "add-module-exports" documentation:
+		 * "webpack doesn't perform commonjs transformation for
+		 * codesplitting. Need to set commonjs conversion."
+		 */
+		['@babel/env', { modules: 'commonjs' }],
```

Comment:
I would prefer that in the babelrc, no?

Reply:
@skjnldsv any quick hints how to do that ? I'm doing this just as a quick fix to get server master CI unblocked

Reply:
okay, I found something https://babeljs.io/docs/en/configuration#babelrcjson

but I'm unsure why these options are attached to a "jsRule" in webpack and whether they'll work the same if I specify the plugin + presets directly in the RC file

Reply:
ok, got it working. 

@skjnldsv please review last commit